### PR TITLE
Updating Tare Page tooltip and Carbon Accounting Contentful

### LINKED ISSUE #218: Investment Visualization Tare Page Correction
## Description

In the DB we are storing **thousands of metric tonnes** of emissions, but this is not reflected in our presentation on the tare page.

image

Firthermore, we source our Exxon data dirrectly from [them](https://corporate.exxonmobil.com/news/reporting-and-publications/advancing-climate-solutions-progress-report?utm_source=google&utm_medium=cpc&utm_campaign=1ECX_GAD_TRAF_OT_Brand_Emissions&utm_content=OT_Brand_Emissions&utm_term=exxonmobil+emissions&gclid=CjwKCAjw8symBhAqEiwAaTA__LMYt2Sl-vOdJ-ghBzfRdHdDDb1FR9EtGFJn_u814nxjxRebWVUxQRoCLlQQAvD_BwE&gclsrc=aw.ds), so we need to update our citation on their tare page

image

## Todo

- [ ] Change tooltip to reflect **thousands of metric tonnes**
- [ ] Change Exxon citation

### PR DESCRIPTION
**Pull request by eeshabarua**
## Summary

This PR updates the Tar page emissions chart tooltip to thousands of metric tons of CO2, and also updates the carbon accounting blurb for Exxon specifically. 

Closes #218 

## Changes

- break up the blurb into an info & citation section
- use both for everything but exon
- for exon just use the info and have a custom citation in the code base

## Screenshots
image

Zoomed out for screenshot:
image
image

### CODE REVIEW

File: src/components/chart/EmissionBarChart.tsx [hunk 1]
```diff
@@ -38,7 +38,8 @@ const EmissionBarChart: FC<EmissionBarChartProps> = ({ emissionData }) => {
   const options = {
     tooltip: {
       y: {
-        formatter: (value: number) => `${value} Metric tons CO2e`,
+        formatter: (value: number) =>
+          `${(value / 1000).toFixed(3)} Thousands of metric tons CO2e`,
```
Comment (eeshabarua) [inline src/components/chart/EmissionBarChart.tsx:42]: i went ahead w 3 decimal precision but this needs confirming

Comment (kennykos) [reply src/components/chart/EmissionBarChart.tsx:42]:
```suggestion
          `${value}k metric tons CO2`,
```
We need just `${value}` as opposed to ` ${(value / 1000).toFixed(3)}` as the value in the DB is stored in the thousand metric tons (sorry, I should have been clearer on this). I am using `k metric tons CO2` to be consistent with the treemap
image

**Review by kennykos — CHANGES_REQUESTED** [review]
Great work! With the small change we should be good to go :)

**Review by kennykos — APPROVED** [review]
Great Work!

**Pull request merged**